quarkus-extension/engine/deployment/src/test/java/org/camunda/bpm/quarkus/engine/test/helper/ProcessEngineAwareExtension.java

```diff
@@ -16,22 +16,22 @@
  */
 package org.camunda.bpm.quarkus.engine.test.helper;
 
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.function.Supplier;
+
```

Comment:
Import order wars. 😂   I usually try to minimally change the order of imports when I touch a file.

Reply:
Sorry about that. IntelliJ optimized them automatically when I refactored the config classes. Would you like me to restore them as before?